test/logging_test.rb

```diff
@@ -0,0 +1,77 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class LoggingTest < Minitest::Test
+      class TestLogger < ActiveSupport::Logger
+        def initialize
+          @file = StringIO.new
+          super(@file)
+        end
```

Comment:
I don't think so. I know that the `ActiveModelSerializers.logger` do his job, so to avoid open the log, parse the log, delete the log and so on. I prefer using a custom logger this way I can have all messages on a easy way. This is the same logic followed by [Active Job](https://github.com/rails/rails/blob/9b663e2679cfdaa2da9bfed0bd4d601309072c22/activejob/test/cases/logging_test.rb#L13-L23)